shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -22,12 +22,26 @@
 import android.text.format.DateFormat;
 import android.util.AttributeSet;
 import android.util.Log;
-import android.view.*;
+import android.view.KeyEvent;
+import android.view.MotionEvent;
+import android.view.Surface;
+import android.view.SurfaceHolder;
+import android.view.SurfaceView;
+import android.view.View;
+import android.view.WindowInsets;
 import android.view.accessibility.AccessibilityManager;
 import android.view.accessibility.AccessibilityNodeProvider;
 import android.view.inputmethod.EditorInfo;
 import android.view.inputmethod.InputConnection;
 import android.view.inputmethod.InputMethodManager;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.ArrayList;
```

Comment:
Are these dependencies added by accident?

Reply:
I ran an import cleanup through the IDE. As far as I know everything in the import list is needed and they're now in the canonical order. But if anything looks wrong, please let me know.

Reply:
I see, so you actually replaced the wildcard imports with a list of classes, and moved some around.

Reply:
I honestly didn't pay much attention to what the IDE did :) But that would seem to be what happened. It's probably a good idea to replace wildcards, anyway, because they tend to obfuscate dependencies at first glance.